vendor/bat-native-ledger/src/transactions_info.cc

```diff
@@ -0,0 +1,94 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/transactions_info.h"
+
+#include "base/json/json_reader.h"
+#include "base/json/json_writer.h"
+#include "base/values.h"
+
+namespace ledger {
+
+TransactionsInfo::TransactionsInfo() :
+    transactions({}) {}
+
+TransactionsInfo::TransactionsInfo(const TransactionsInfo& info) :
+    transactions(info.transactions) {}
+
+TransactionsInfo::~TransactionsInfo() = default;
+
+const std::string TransactionsInfo::ToJson() const {
+  base::Value dictionary(base::Value::Type::DICTIONARY);
+
+  base::Value list(base::Value::Type::LIST);
+  for (const auto& transaction : transactions) {
+    base::Value transaction_dictionary(base::Value::Type::DICTIONARY);
+
+    transaction_dictionary.SetKey("timestamp_in_seconds",
+        base::Value(std::to_string(transaction.timestamp_in_seconds)));
+
+    transaction_dictionary.SetKey("estimated_redemption_value",
+        base::Value(transaction.estimated_redemption_value));
+
+    list.GetList().push_back(std::move(transaction_dictionary));
+  }
+
+  dictionary.SetKey("transactions", base::Value(std::move(list)));
+
+  // Write to JSON
+  std::string json;
+  base::JSONWriter::Write(dictionary, &json);
+
+  return json;
+}
+
+bool TransactionsInfo::FromJson(
```

Comment:
we should be using the same approach that we are using in ledger.cc.  see `bool Grant::loadFromJson(const std::string& json)` for example

Reply:
Confirmations has been refactored to use base::JSONReader/Writer and not RapidJSON, which was discussed a while ago, this code has been taken from Confirmations and I would recommend we keep this as all other code should eventually be refactored to use base where possible

Reply:
answered above